submissions/dummy_submission/the-nuclear-options.md

```diff
@@ -0,0 +1,150 @@
+
+* **Minimize heat pipe usage** - The game's heat manager was found to be the other big UPS eater (around 20% of an optimized setup). The number of heat connections is main factor to consider, meaning that heat pipes should be as short as possible and be only one tile thick.
+
+* **Refuel reactor arrays using bots or clocked belts** - While bots may sounds less UPS-efficient than belts, the issue with using belts is inserter tracking, which is when the inserters try to catch moving items on belts. This issue occurs a lot when you have a long belt servicing many reactors. It causes an extra 6% of UPS overhead to an optimized setup. The way around the issue is to use bots to deliver fuel cells. Alternatively, you can use a transport belt that is not looping and inserters that are "clocked", meaning that they are timed using circuit networks such that they all take from the belt at once and then wait for the belt to fill up before they are allowed to take again.
+
+* **Minimize fluid entity usage, particularly pumps** - In an optimized setup, the fluid system accounts for less than 1% of UPS drain. On the other hand, a setup with pressurized steam pipes (one pump per two exchangers) had an extra UPS overhead of 15%. Hence it is preferable to use very few pumps, while afterwards one may consider removing storage tanks (starting from steam storage).
+
+* **Prioritize optimizing what takes up the most time** - In Factorio, the heat manager, the fluid manager, and the electric network are simulated in parallel. Therefore, the slowest of these three factors causes the UPS bottleneck for the group while the other two have little to no impact. You can check this by opening the debug settings with the `F4` key and selecting "show-time-usage" and looking at which factor lists the largest numbers for update times. Based on this you can determine what to add or remove from your designs. For example, if the fluid manager costs comparatively little time, you can add more steam storage without worry.
+
+* **If you want to build a megabase, transition to solar power eventually** - In the end nuclear cannot compete with solar, which has simpler calculations and excellent performance optimizations. However, since solar power is more expensive than nuclear power in terms of setup costs and space usage, starting your large scale power production with a few GW of nuclear energy is still worth the trouble since the impact at that scale is small.
```

Comment:
```suggestion
* **If you want to build a megabase, transition to solar power eventually** - In the end nuclear cannot compete with solar, which has simpler calculations and excellent performance optimizations. However, since solar power is more expensive than nuclear power in terms of setup costs and space usage, starting your large-scale power production with a few GW of nuclear energy is still worth the trouble since the impact at that scale is small.
```

Reply:
Maybe `in terms of setup costs` -> `in terms of setup cost` ?

Reply:
It is plural because the cost is in terms of various raw resources. However, if it is too distracting this way, having it singular works too. I trust your judgement.

Reply:
Meanwhile, there is a concern about "the slowest of these three factors causes the UPS bottleneck for the group while the other two have little to no impact" based on https://discord.com/channels/745990677606826005/748304365831323830/945482167037882398 . I asked flame_Sla to review how to rephrase this.